test/firrtl/errors.mlir

```diff
@@ -122,3 +122,16 @@ firrtl.circuit "Foo" {
     %a = firrtl.reginit %clk, %reset, %zero {name = "a"} : (!firrtl.clock, !firrtl.uint<1>, !firrtl.uint<1>) -> !firrtl.flip<uint<1>>
   }
 }
+
+// -----
+
+firrtl.circuit "Foo" {
+
+  // expected-error @+2 {{'firrtl.extmodule' op attribute 'defname' with value 'Bar' conflicts with the name of another module in the circuit}}
+  // expected-note @+2 {{previous module declared here}}
```

Comment:
Sure nit but I'd move the note line to being just above the `@Bar` line and use `@+1` instead of +2.

Reply:
Fair. I went back and forth with this. Will update.